Sources_v3/StreamChat/Controllers/ChannelController/ChannelController.swift

```diff
@@ -508,8 +508,13 @@ public extension _ChatChannelController {
             channelModificationFailed(completion)
             return
         }
+
+        guard let currentUserId = client.currentUserId else {
+            callback { completion?(ClientError.CurrentUserDoesNotExist()) }
+            return
+        }
         
-        updater.hideChannel(cid: cid, userId: client.currentUserId, clearHistory: clearHistory) { error in
+        updater.hideChannel(cid: cid, userId: currentUserId, clearHistory: clearHistory) { error in
```

Comment:
This is strange 🤔 Do we really need `currentUserId` to hide a channel? Maybe the `HideChannelRequest` doesn't have to contain the current user id? The backend should get the current user id from the token, as it does for all other requests.

Reply:
removed and provided the reason in bcab130e7e58c4a2bd87df2c991488555d208e41